src/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(person, date, self)
+  end
```

Comment:
- [ ] Please note, like the previous reviewer said, that you have missed the order of the attributes which you need to create a new rental.
``` ruby
Rental.new(date, self, person)
```